### ISSUE TITLE
No module named request

### ISSUE THREAD

**Issue by ParadoxalGamer** [k4m4/kickthemout#325]
## Checklist for submitting an issue to `KickThemOut`:

- [x] I have carefully read the [README](https://github.com/k4m4/kickthemout/blob/master/README.rst) file and haven't managed to resolve my issue.
- [x] I have searched the [issues](https://github.com/k4m4/kickthemout/issues?utf8=%E2%9C%93&q=is%3Aissue) of this repo and believe that this is not a duplicate.
- [x] I am running the latest version of KickThemOut.

 

- **OS name & version**: Kali Linux 2018.2 amd64
- **Python version**: 3.4
- **Scapy version**: 3-0.25
- **Nmap version**: 0.6.1
- **Link of [Gist](https://gist.github.com/)**: https://gist.github.com/ParadoxalGamer/b5e7315dc21453755a0c46bbdc57f4b3

 The soft tell me I don't have the request module but it's not in the requirements, what can I do to fix the problem ? (please remember i'm french and this is because I can write with some errors<br/ >

**Comment by AspecTT** [comment]
I'm having the same issue.
I read the requirements needed, so I went to install them. I was able to get python-scapy installed, so when I ran sudo python kickthemout.py, it gave me an error saying "no module named nmap." Okay, I installed python-nmap. Boom. I run sudo python kickthemout.py again, and it said "no module named request". Oh, I'll install python netifaces. I installed it, and when I ran sudo python kickthemout.py it kept giving me the error above.
Any ideas if I did something wrong?
